src/styles/index.scss

```diff
@@ -1,3 +1,125 @@
+@import './variables';
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  display: flex;
+}
+
+.card {
+  width: 200px;
+  height: 406px;
```

Comment:
do not hardcode height. content card should make height for card.
 check design. card should have border. total card width in design 200px, total height - 408px
```suggestion
```